.github/workflows/test.yaml

```diff
@@ -44,9 +44,6 @@ jobs:
         # sonarcloud needs full git histories
         with:
           fetch-depth: 0
-      - name: Build ota-test_base docker image
-        run: |
-          docker compose -f docker/test_base/docker-compose_tests.yml build tester-${{ matrix.test_base }}
```

Comment:
(nits) We don't have to include the change into this PR.
It will be useful to add "base image creation" pipeline.

Reply:
Currently there is no plan to implement such pipeline, as we expect that the test base container images will be stable, unless we change the specification.

Even we have a pipeline to create the test base docker image, it will not be in the `test.yaml`, and will not be executed by test CI. It should be a standalone CI.